test/core/math/quat.test.mjs

```diff
@@ -548,6 +561,56 @@ describe('Quat', function () {
 
     });
 
+    describe('#setFromDirections()', function () {
+
+        it('set the identity quaternion from equal directions', function () {
+            const v1 = new Vec3(1, 0, 0);
+            const v2 = new Vec3(1, 0, 0);
+
+            const q1 = new Quat().setFromDirections(v1, v2);
+            expect(q1.equals(Quat.IDENTITY)).to.be.true;
+
+
+            const v3 = new Vec3(0, 0, 0);
+            const v4 = new Vec3(0, 0, 0);
+
+            const q2 = new Quat().setFromDirections(v3, v4);
+            expect(q2.equals(Quat.IDENTITY)).to.be.true;
+        });
+
+        it('set a quaternion from different directions', function () {
+            const epsilon = 0.00001;
+
+            const v1 = new Vec3(1, 0, 0);
+            const v2 = new Vec3(0, 1, 0);
+
+            const q1 = new Quat().setFromDirections(v1, v2);
+            const q2 = new Quat().setFromEulerAngles(0, 0, 90);
+
+            expect(q1.equalsApprox(q2, epsilon)).to.be.true;
```

Comment:
The point about having a custom method is that you can easily set default values, which you don't need to re-define here any longer, so we have shorter and fewer lines

Reply:
Done :-)